.evergreen.yml

```diff
@@ -772,12 +782,8 @@ tasks:
   ###
   # PACKAGING
   ###
-  - name: package_and_upload_artifact_macos_x64
+  - name: package_and_upload_artifact_darwin_x64
     depends_on:
-      - name: check
-        variant: darwin
-      - name: ".unit-test"
-        variant: darwin
```

Comment:
This was annoying during releases because the packaging step didn’t start until *all* tests and in particular their possible re-runs had succeeded, and getting rid of this is just kind of convenient. Let’s just build packages and test them regardless of the success of other tests.

The dependencies here are moved to the draft task.